docker/Dockerfile

```diff
@@ -8,51 +8,46 @@ RUN conda update conda && \
 
 FROM base as dependencies
 
-ENV PATH="/opt/venv/bin:$PATH"
```

Comment:
These dockerfile changes make sense to me, but I am surprised it was even working with the previous version of the Dockerfile ? 🤔